js/views/participantlistview.js

```diff
@@ -131,8 +131,10 @@
 			},
 			onRender: function() {
 				this.$el.find('.avatar').each(function() {
-					var element = $(this);
-					if (element.data('displayname').length) {
```

Comment:
this compared the display name instead of the user id and also data returns "numbers" as "int" and int.length is undefined....